Assets/SEE/Controls/SEEInput.cs

```diff
@@ -18,6 +20,32 @@ public static class SEEInput
         /// forward.
         /// </summary>
         public static bool KeyboardShortcutsEnabled { set; get; } = true;
+        private static readonly Dictionary<string, KeyCode> bindings = BindingNamesAndKeys();
+
+        /// <summary>
+        /// Updates the bindings.
```

Comment:
What does this updating mean precisely?

Reply:
There are two synchronised IDictionarys, the one in KeyBindings and another one in SEEInput, which only has the Keys and binding names. I did this, because I thought it would be a bad idea to make the bindings IDictionary in KeyBindings.cs public. But since we can change the bindings at runtime now, the IDictionary in SEEInput needs to get updated, when the user changes a key.